cfgov/v1/models/snippets.py

```diff
@@ -79,3 +81,83 @@ class RelatedResource(index.Indexed, models.Model):
 
     def __str__(self):
         return self.title
+
+
+@python_2_unicode_compatible
+@register_snippet
+class GlossaryTerm(index.Indexed, models.Model):
+    term_en = models.CharField(
+        max_length=255,
+    )
+    definition_en = RichTextField(
+        null=True,
+        blank=True
+    )
+    anchor_en = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True
+    )
+    answer_page_en = models.ForeignKey(
+        'ask_cfpb.AnswerPage',
+        related_name='glossary_terms',
+        null=True,
+        blank=True
+    )
+    term_es = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True
+    )
+    definition_es = RichTextField(
+        null=True,
+        blank=True
+    )
+    anchor_es = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True
+    )
+    answer_page_es = models.ForeignKey(
+        'ask_cfpb.AnswerPage',
+        related_name='glossary_terms_es',
+        null=True,
+        blank=True
+    )
+    portal_topic = ParentalKey(
+        'v1.PortalTopic',
```

Comment:
Do we need to allow for the possibility that a glossary term could be associated with multiple portal topics?

Reply:
@virginiacc I don't think so -- there's a few glossary term names like "Amortization" that appear in multiple topics, and they have their own definitions etc.  If they really want to duplicate content across portals I think it might just be easiest to create a new snippet and copy and paste but I'd say we shouldn't design for it. 

@sonnakim any thoughts?

Reply:
Ok, that makes sense. We can always revisit if there's a need for multiple topics down the road.